getting_started.rst

```diff
@@ -1,170 +1,160 @@
 .. _getting_started:
 
+
 Getting started
 ===============
 
-As an example, let's start with one of the most popular C++ libraries: POCO_.
-Conan **works with any build system** and it does not depend on CMake, though we will use CMake for
-this example for convenience.
+Let's start with an example using one of the most popular C++ libraries: POCO_. For convenience purposes we'll use CMake.
+Keep in mind that Conan **works with any build system** and does not depend on CMake.
 
 .. _POCO: https://pocoproject.org/
 
+
 A Timer using POCO libraries
 ----------------------------
 
 First, let's create a folder for our project:
 
 .. code-block:: bash
 
-    $ mkdir mytimer
-    $ cd mytimer
-
+   $ mkdir mytimer
```

Comment:
Please do not change indentation of code-blocks. Not necessary, and introduce a lot of noise for the review. Thanks!